packages/@tinacms/graphql/src/database/level.ts

```diff
@@ -8,30 +8,21 @@ import {
   AbstractSublevelOptions,
 } from 'abstract-level'
 
-export type Level = AbstractLevel<
-  Buffer | Uint8Array | string,
-  string,
-  Record<string, any>
->
+export type Level = AbstractLevel<any, string, Record<string, any>>
```

Comment:
I'm not sure about this change. I think the d1 abstract-level should work without making this change. Have a look at https://github.com/tinacms/upstash-redis-level/blob/main/src/index.ts